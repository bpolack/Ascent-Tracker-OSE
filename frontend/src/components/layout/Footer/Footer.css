.CopyrightContainer {
	padding: 15px 30px;
	text-align: center;
	color: var(--ascent-light);
}