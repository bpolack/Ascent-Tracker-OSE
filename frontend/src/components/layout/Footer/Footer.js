import React, { Component } from 'react'

export class Footer extends Component {
	render() {
		return (
			<footer className="relative bg-ascent-std text-ascent-std text-sm text-center px-8 py-4">
				<span>Copyright Â© Ascent Time Tracker | Made with ðŸŒ® by <a href="https://polackdevelopment.com">Braighton Polack</a></span>
			</footer>
		)
	}
}

export default Footer
