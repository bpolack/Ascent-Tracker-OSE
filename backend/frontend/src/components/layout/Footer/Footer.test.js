import React from 'react';
import { shallow } from 'enzyme';
import { Footer } from './Footer';

describe('Footer', () => {
	it("Renders correctly without crashing", () => {
		const wrapper = shallow(<Footer />);
		expect(wrapper).toMatchSnapshot();
	});
});