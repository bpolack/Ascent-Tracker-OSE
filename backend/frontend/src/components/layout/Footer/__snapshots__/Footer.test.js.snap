// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Renders correctly without crashing 1`] = `ShallowWrapper {}`;
